docs/develop/dialog-best-practices.md

```diff
@@ -36,9 +36,9 @@ For best practices in dialog box design, see [Dialog boxes in Office Add-ins](..
 
 ### Handling pop-up blockers with Office on the web
 
-Attempting to display a dialog box while using Office on the web may cause the browser's pop-up blocker to block the dialog box. Office on the web has a feature that enables your add-in's dialog boxes to be an exception to the browser's pop-up blocker. When your code calls the `displayDialogAsync` method, then Office on the web will open a prompt similar to the following:
+Attempting to display a dialog box while using Office on the web may cause the browser's pop-up blocker to block the dialog box. Office on the web has a feature that enables your add-in's dialog boxes to be an exception to the browser's pop-up blocker. When your code calls the `displayDialogAsync` method, then Office on the web will open a prompt similar to the following.
```

Comment:
Maybe combine "Office on the web has a feature" and "When your code calls" for something like 
"However, using the `displayDialogAsync` method will ensure that Office on the web will ensure your dialog boxes are an exception and open a prompt similar to..."